fe/fe-core/src/main/java/com/starrocks/staros/StarMgrServer.java

```diff
@@ -261,4 +262,26 @@ public long getMaxJournalId() {
     public long getReplayId() {
         return getJournalSystem().getReplayId();
     }
+
+    public static String[] getBucketAndPrefix() {
+        int index = Config.aws_s3_path.indexOf('/');
```

Comment:
can consider use `Config.aws_s3_path.split("/", 2)`

Reply:
hard to use ...